addons/website/models/ir_ui_view.py

```diff
@@ -240,6 +240,26 @@ def _get_top_level_view(self):
         self.ensure_one()
         return self.inherit_id._get_top_level_view() if self.inherit_id else self
 
+    def get_view_hierarchy(self):
+        self.ensure_one()
+        top_level_view = self._get_top_level_view().with_context(active_test=False)
```

Comment:
Could you remove that `_get_top_level_view` util now that we don't have to do that in xml anymore?

We generally just do:
```py
        top_level_view = self
        while top_level_view.inherit_id:
            top_level_view = top_level_view.inherit_id
```